src/main/java/it/gov/pagopa/payhub/activities/dao/OrganizationInstallmentTypeDao.java

```diff
@@ -0,0 +1,10 @@
+package it.gov.pagopa.payhub.activities.dao;
+
+import it.gov.pagopa.payhub.activities.dto.OrganizationInstallmentTypeDTO;
+
+import java.util.List;
+
+public interface OrganizationInstallmentTypeDao {
```

Comment:
let's DAO's and model's name to the actual design model:
https://excalidraw.com/#json=vdAn67Yr1Qh-FbHvBiMVr,ZqMASxIRoY8DJxbVNsMajQ
```suggestion
public interface DeptPositionTypeOrgDao {
```